test/test_dependencies.py

```diff
@@ -0,0 +1,20 @@
+from mock import Mock, call, patch
+import pytest
+
+from nameko.dependencies import Config
+
+
+class TestConfig:
+
+    @pytest.yield_fixture(autouse=True)
+    def patch_container(self):
+        with patch.object(Config, 'container'):
+            yield
```

Comment:
This is quite an "internal" looking test. For such a simple dependency provider it would be better just to demonstrate that it works.

You could simply duplicate the test you have above to verify your example. That is "real-world" usage, therefore less brittle, and would show an example service and how to use it next to each other.